src/pug/body.pug

```diff
@@ -0,0 +1,6 @@
+div.pf-c-background-image
+div.pf-c-page#page
+    include header.pug
+    include sidebar.pug
+    main.pf-l-page__main#root(role='main')
+        include footer.pug
```

Comment:
Hrrmmm this is tricky.
Right now Chrome.js is incorrectly included in sidebar.html.

We probably want to correct this w/o causing breakage for current consumers.

Can we
- duplicate sidebar (sidebar2)
- remove the chrome.js include in sidebar2
- use sidebard2 in body.html
- "after.html" or "final.html"
- put the chrome.js include in after.html
- put after.html on the bottom
- put a script console.log('Deprecation warning') in the old sidebar.html

I think that leaves unupdated applications working and lets updated apps transition to the new unbroken stuff.

Reply:
I assumed we'd want more than just "Deprecation warning", so I made it a bit more descriptive:
`Deprecation warning: Please update your usage of Chrome.`